Suggested changes

miguelsosa opened the pull request:
Hey - check these suggestions. 

Don't just plop down my changes - I have some comments in there for you to re-write.

miguelsosa committed:
Suggested changes

MichaelDimmitt commented:
@miguelsosa, thanks for the interest. In the future you may want to start conversations on the pull request instead of making a comment in the readme.md! It allows you you to make comments about parts of the code and to have replies and discussions in those locations instead of putting the comments directly inside the code. 

That aside, there are some readme changes here that are good and some changes I am going to improve and bring up for discussion. Thanks for the commit!

MichaelDimmitt committed:
responding to Miguel's changes check extended commit description.

Changes implemented on this commit:
1) change to title
2) format correcting first paragraph for further discussion.
3) cleanup line and correct grammar for line 17.
4) attempt to fix formatting for bullet 5 on additional features.

MichaelDimmitt commented:
@miguelsosa, Thanks for looking at the readme and suggesting changes. With an additional commit, I modified your suggestions. Give the readme another look-over and if you are okay with it I think its ready to merge. 

Warning, if you had unsaved changes they are going to be overridden by my commit when you pull.
cheers, Michael Dimmitt
